askama_shared/src/path.rs

```diff
@@ -52,6 +54,36 @@ pub fn template_dir() -> PathBuf {
     path
 }
 
+#[derive(Deserialize)]
+struct Config {
+    dirs: Option<Vec<String>>,
+}
+
+pub fn template_dirs() -> Vec<PathBuf> {
+    let root = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+    let askama = root.join("askama.toml");
```

Comment:
I would like to move `"askama.toml"` into a constant, maybe named `CONFIG_FILE_NAME`. Then, please move the code to instantiate a `Config` into a `Config::from_file()` method (which doesn't need any arguments).

Also, here and in other places, please don't override the configuration if it provides an empty list. In that case, it should probably just do nothing (this would allow users to enforce the use of `source`-based templates only, I guess).